crates/stages/src/test_utils/runner.rs

```diff
@@ -78,7 +82,7 @@ pub(crate) trait UnwindStageTestRunner: StageTestRunner {
             provider.commit().expect("failed to commit");
             tx.send(result).expect("failed to send result");
         });
-        Box::pin(rx).await.unwrap()
+        async { rx.await.unwrap() }
```

Comment:
The body of the entire function should be wrapped in `async move {}` to preserve the same semantics. Although I don't think this matters much in this case